plugins/BEdita/Core/src/Model/Behavior/UniqueNameBehavior.php

```diff
@@ -82,6 +82,9 @@ public function uniqueName(EntityInterface $entity)
         if (empty($uname)) {
             $uname = $this->generateUniqueName($entity);
         }
+        else {
```

Comment:
* coding style =>  use `} else {` - see code sniff failure in Travis CI 
* we should add some test to `UniqueNameBehaviorTest` 
* we have a problem with  `UsersTable::DELETED_USER_PREFIX` - `__deleted-` on CI tests in case of user anonymoization -- we could use a different uname prefix in that case... (see https://travis-ci.org/github/bedita/bedita/jobs/703211808)

Reply:
I'll add some tests and fix the code style.
About the third point, we can allow underscore character by using the options parameter of `Text::slug()`